bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/console.xml

```diff
@@ -42,6 +42,10 @@
 		<label>Beep Console</label>
 		<description>Send a beep command to this/all console(s).</description>
 		<category>SoundVolume</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>SoundVolume</tag>
```

Comment:
Although sending beeps will produce sound, I'm not sure if it's right to tag it like a sound volume.